src/knl/knl_vadv_variant_ij_blocked.cpp

```diff
@@ -25,11 +25,9 @@ namespace platform {
             const int isize = this->isize();
             const int jsize = this->jsize();
             const int ksize = this->ksize();
-            constexpr int istride = 1;
+            const int istride = this->istride();
```

Comment:
Not sure if we can do that without losing ICC’s vectorization, will check…

Reply:
It actually kills the KNL performance…

Reply:
kij layout in this function behaves better than other layouts on ARM (we cant vectorize in any layout here). But still, this was not where we got the best vadv performance.

Reply:
It is (was) the fastest for KNL, though. So let’s just keep the constexpr thingy?

Reply:
Okay, done